public/app/plugins/panel/graph/graph.js

```diff
@@ -352,6 +352,45 @@ function (angular, $, moment, _, kbn, GraphTooltip) {
           };
         }
 
+        //Override min/max to provide more flexible autoscaling
+        function autoscaleSpanOverride(yaxis, data, options) {
+          var m, op, num, precision;
+          if (yaxis.min != null && data != null) {
+            m = yaxis.min.match(/([<=>~]*)\W*(\d+(\.\d+)?)/);
```

Comment:
@lpalm I have a bit of a difficult time figuring out where <= and >= would be useful? Could you explain one case for me? 

I agree that the regex could be a lot better. I think we need \W* because \W would enforce a white space.